src/praisonai/praisonai/integrations/compute/tenki.py

```
@@ -0,0 +1,312 @@
+                "sudo apt-get update -y && sudo apt-get install -y python3-pip; fi && "
+                f"pip3 install -q --break-system-packages {' '.join(pip_pkgs)}"
+            )
+            logger.info("[tenki_compute] installing pip: %s", pip_pkgs)
+            try:
+                result = sandbox.exec("bash", "-lc", cmd, timeout=300)
+                if result.exit_code != 0:
+                    logger.warning(
+                        "[tenki_compute] pip install failed: %s",
+                        (result.stderr or b"").decode(errors="replace"),
+                    )
+            except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)